lib/supabase/get-user.ts

```diff
@@ -12,12 +14,29 @@ import { createClient } from "./server";
 export const getUser = async () => {
 	const supabase = await createClient();
 
-	const { data, error } = await supabase.auth.getUser();
-	if (error != null) {
-		throw error;
-	}
-	if (data.user == null) {
-		throw new Error("No user returned");
-	}
-	return data.user;
+	const getUserFunc = async () => {
+		const { data, error } = await supabase.auth.getUser();
+
+		if (error != null) {
+			throw error;
+		}
+		if (data.user == null) {
+			throw new Error("No user returned");
+		}
+		return data.user;
+	};
+
+	const retryCount = 5;
```

Comment:
nits:
If there is any context behind setting the value to 5, I would appreciate it if you could include it in a comment. 
Even if it's just a fixed value, I would appreciate it if you could indicate that as well.

Reply:
There is no specific reason.
So I'll add comment to indicate that.